src/mrnet/network/reaction_network.py

```diff
@@ -1163,7 +1330,7 @@ def remove_node(self, node_ind: List[int]):
             for node in nodes:
                 if self.graph.nodes[node]["bipartite"] == 1:
                     reactants = node.split(",")[0].split("+")
-                    reactants = [reac.replace("PR_", "") for reac in reactants]
+                    reactants = [reac.replace("PR_", "") for reac in reactants]  # flag
```

Comment:
Can this line just be removed? It seems like the previous line should work as intended given "A+B,C" or "A,B"